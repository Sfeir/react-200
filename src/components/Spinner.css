.Spinner {
  position: absolute;
  top: calc(50% - 20px);
  left: calc(50% - 20px);
}

.Spinner .spinner-layer {
  border-color: #0168AB;
}
