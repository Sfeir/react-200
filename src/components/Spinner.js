import React from 'react';
import './Spinner.css';

const Spinner = () => (
  <div className="Spinner">
    <div className="preloader-wrapper big active">
      <div className="spinner-layer">
        <div className="circle-clipper left">
          <div className="circle" />
        </div>
        <div className="gap-patch">
          <div className="circle" />
        </div>
        <div className="circle-clipper right">
          <div className="circle" />
        </div>
      </div>
    </div>
  </div>
);

export default Spinner;
