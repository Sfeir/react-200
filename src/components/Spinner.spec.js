import React from 'react';
import { create } from 'react-test-renderer';
import Spinner from './Spinner';

describe('Spinner', () => {
  test('snapshot', () => {
    const out = create(<Spinner />).toJSON();
    expect(out).toMatchSnapshot();
  });
});
