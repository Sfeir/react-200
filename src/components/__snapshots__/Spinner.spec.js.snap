// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snapshot 1`] = `
<div
  className="Spinner"
>
  <div
    className="preloader-wrapper big active"
  >
    <div
      className="spinner-layer"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>
</div>
`;
